src/index.js

```diff
@@ -142,17 +142,30 @@ export function app(state, actions, view, container) {
     return node ? node.key : null
   }
 
+  function eventListener(event) {
+    return event.target.events[event.type](event)
+  }
+
   function updateAttribute(element, name, value, oldValue, isSVG) {
     if (name === "key") {
     } else if (name === "style") {
       for (var i in clone(oldValue, value)) {
         element[name][i] = value == null || value[i] == null ? "" : value[i]
       }
     } else {
-      if (
-        typeof value === "function" ||
-        (name in element && name !== "list" && !isSVG)
-      ) {
+      if (name[0] === "o" && name[1] === "n") {
+        if (!element.events) {
+          element.events = {}
+        }
+        element.events[(name = name.slice(2))] = value
```

Comment:
What do you think of adding `.toLowerCase()` for supporting the common variations like `onClick`?

Reply:
Doesn't this line introduce memory leak? I do not see `element.events` cleanup...

Reply:
@frenzzy Good question. We are doing the clean up here:

```js
element.events[(name = name.slice(2))] = value
```

...when `value` is null or undefined, as well as calling `removeEventListener`.